syntax = "proto3";

package profile;

service ProfileService {
  rpc ChangeNickname (ProfileRequest) returns (ProfileReply) {}
}

message ProfileRequest {
  string id = 1;
  string nickname = 2;
  string email = 3;
}

message ProfileReply {
  string message = 1;
}

